app/controllers/user.js

```diff
@@ -5,7 +5,9 @@ const bcrypt = require('bcrypt'),
   salt = bcrypt.genSaltSync(saltRounds),
   errors = require('../errors'),
   { validateUser } = require('./validations'),
-  User = require('../models').users;
+  User = require('../models').users,
+  { encoder, HEADER } = require('./../services/jwt'),
```

Comment:
The service name could be not coupled to implementation, you can name this like `authentication` or something better